test/013_oracles_composite.ts

```diff
@@ -1,24 +1,30 @@
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address'
 import { constants, id } from '@yield-protocol/utils-v2'
 const { WAD } = constants
-import { ETH, DAI, USDC } from '../src/constants'
+import { ETH, DAI, USDC, WSTETH, STETH } from '../src/constants'
 
 import ChainlinkMultiOracleArtifact from '../artifacts/contracts/oracles/chainlink/ChainlinkMultiOracle.sol/ChainlinkMultiOracle.json'
 import CompositeMultiOracleArtifact from '../artifacts/contracts/oracles/composite/CompositeMultiOracle.sol/CompositeMultiOracle.json'
 import ChainlinkAggregatorV3MockArtifact from '../artifacts/contracts/mocks/oracles/chainlink/ChainlinkAggregatorV3Mock.sol/ChainlinkAggregatorV3Mock.json'
 import DAIMockArtifact from '../artifacts/contracts/mocks/DAIMock.sol/DAIMock.json'
 import USDCMockArtifact from '../artifacts/contracts/mocks/USDCMock.sol/USDCMock.json'
 import WETH9MockArtifact from '../artifacts/contracts/mocks/WETH9Mock.sol/WETH9Mock.json'
-
+import STETHMockArtifact from '../artifacts/contracts/mocks/STETHMock.sol/STETHMock.json'
```

Comment:
This is so weird because in the other test it doesn't have a problem finding it!

Reply:
Sometimes, after a contract being renamed, the local artifacts still have files that make it work, but the CI copy doesn't work because it's fresh. Removing the artifacts folder in the local copy sometimes reproduces the problem.